tests/plugins/elementspath/elementspath.js

```diff
@@ -28,6 +28,35 @@
 		'test widget mockup element path': function() {
 			this.editorBot.setHtmlWithSelection( bender.tools.getValueAsHtml( 'testWidgetSample' ) );
 			this.assertPath( 'div,figcaption' );
+		},
+
+		// (#438)
+		'test focusing toolbar': function() {
+			var editor = this.editor,
+				commandSpy = sinon.spy( editor, 'execCommand' );
+
+			this.editorBot.setHtmlWithSelection( '<b>f^oo</b>' );
+
+			editor.ui.space( 'path' ).getFirst().$
+				.onkeydown( { keyCode: 121, altKey: true } ); // ALT + F10
+
+			commandSpy.restore();
+			assert.isTrue( commandSpy.calledWith( 'toolbarFocus' ) );
+		},
+
+		// (#438)
+		'test focusing editor': function() {
```

Comment:
I don't know if it's worth it. We will remove very small duplication here because almost half of the test will have to be still configured by arguments (assertion, keystroke, and spy).

Of course, we could extract test code into 2 helper functions shared between `toolbar` and `elementspath` test suites, but is it worth effort for such small duplication inside unit tests?